devel/swiftlint/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           xcodeversion 1.0
+
+github.setup        realm swiftlint 0.52.4
+
```

Comment:
please add the "github.tarball_from" line and set it to "releases"(if upstream provides those) or use "archive". Making this change will require you to update the "checksums".